wavefront/server/modules/plugins_module/plugins_module/controllers/datasource_controller.py

```
@@ -532,6 +523,48 @@ async def insert_rows_json(
 
 
+@datasource_router.patch('/v1/datasources/{datasource_id}/resources/{resource_id}')
+@inject
+async def update_rows_json(
+    _: Request,
+    datasource_id: str,
+    resource_id: str,
+    update_rows_json_payload: UpdateRowsJsonPayload,
+    response_formatter: ResponseFormatter = Depends(
+        Provide[CommonContainer.response_formatter]
+    ),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)